static/django_templates/base/index.html

```diff
@@ -1,4 +1,4 @@
-{% load staticfiles %}
+{% load staticfiles compress %}
```

Comment:
I intentionally wanted to disable compressor from settings for project if we are not using it rather than removing compress everywhere. 

If you have a fork, we can enable it again.